src/box/call.c

```diff
@@ -160,6 +163,10 @@ box_run_on_call(enum iproto_type type, const char *expr, int expr_len,
 	trigger_run(&box_on_call, &ctx);
 }
 
+bool
+box_lua_call_runtime_priv_check(const char *uname, uint32_t uname_len,
```

Comment:
Nit:
- Please make this function `static`.
- Let's name it `box_lua_call_runtime_priv_is_granted`. A function called `FOO_check` usually returns 0 or -1 in Tarantool.

Reply:
Renamed.